tests/e2e/test_e2e_template.py

```diff
@@ -131,21 +139,21 @@ def _run_make_setup_test(tmp_path: Path) -> None:
     )
     job_id = parse_job_id(out)
 
-    expected_string = "Hello World!"
-    tmp_path.mkdir(exist_ok=True)
-    out_file = (tmp_path / "out").absolute()
```

Comment:
I was so stupid to create the file's temp directory locally and expect it to be present in the job :joy: